modules/integration_aws-backup/conf/00-aws-backup-failed.yaml

```diff
@@ -0,0 +1,22 @@
+module: AWS Backup
+name: failed
+id: backup_failed
+
+transformation: ".min(over='1h')"
```

Comment:
I am not sure using a transformation is useful here, we do not need delay for triggering alert on error.

expect if you think it is possible an error was automatically resolved and want to prevent fire alert during a short period to let him the time ?

Reply:
Addressed: https://github.com/claranet/terraform-signalfx-detectors/pull/457/files#diff-0eec7b99d6548a77ff0d378c3db0de6cae5c0527a186aeab8ed197d0e80904a3R5